src/plugins/chart_expressions/expression_xy/public/components/xy_chart.tsx

```diff
@@ -27,6 +28,7 @@ import {
   Placement,
   Direction,
   XYChartElementEvent,
+  Tooltip as TooltipSpec,
```

Comment:
To avoid confusion, is better to keep elastic-charts components name and change the other `Tooltip` import to be something like `import {Tooltip as CustomToolti} from './tooltip';`

Reply:
Fixed https://github.com/elastic/kibana/pull/148372/commits/455999b7f6c9fbde4025a4c96b6881de08b55e33